crates/lib/src/bootc_composefs/boot.rs

```diff
@@ -474,7 +474,8 @@ pub(crate) fn setup_composefs_bls_boot(
             let boot_digest = compute_boot_digest(usr_lib_modules_vmlinuz, &repo)
                 .context("Computing boot digest")?;
 
-            let default_sort_key = "1";
+            let default_sort_key = bootloader.default_sort_key();
```

Comment:
Hmm I think it'd be cleaner if computed all the BLS entries without writing them, and then walk over all of them and reverse the order if we detect grub or so?